content/vendor/guides/airgapped-cluster.md

```diff
@@ -0,0 +1,184 @@
+
+### Airgap Bundle
+
+To download the `.airgap` bundle, go to `[App Name] -> Channels -> [Channel Name] -> Release History`.
+
+![Airgap Channels](/images/guides/kots/airgap-channels.png)
+
+* Click on `Build` to build the `.airgap` bundle.
+
+* Once you see `Download Bundle`, click to download the `.airgap` bundle.
+
+![Airgap Download Bundle](/images/guides/kots/airgap-download-bundle.png)
+
+### Airgap License
+
+To download the airgap license, go to `[App Name] -> Customer -> [Customer Name]`.
+
+![Airgap Customers](/images/guides/kots/airgap-customers.png)
+
+* Select `License options -> Airgap Download Enabled` and `Save Changes`.
+
+* Click on `Download license` to re-download the airgapped enabled yaml license.
+
+![Airgap Download License](/images/guides/kots/airgap-download-license.png)
+
+## Embedded kURL Cluster
+
+Installing KOTS app into an embedded kURL kubernetes cluster needs a few additional steps.
+
+### Download and Install kURL bundle
+
+In addition to the `.airgap` bundle and yaml license file, you also need the kURL airgap bundle which includes the open-source components such as Docker, Kubernetes, KOTS Adm, etc.
+
+Lets download this bundle onto a jumpbox which has internet access.
+
+* To download the kURL bundle for the `Stable` channel.
+```shell
+export REPLICATED_APP=<app_slug>
+curl -LS https://k8s.kurl.sh/bundle/${REPLICATED_APP}.tar.gz -o ${REPLICATED_APP}.tar
+```
+
+* To download the kURL bundle for all other channels, use the suffix `-<channel_name>`.
+```shell
+export REPLICATED_APP=<app_slug>
+export REPLICATED_CHANNEL=<channel_name>
+curl -LS https://k8s.kurl.sh/bundle/${REPLICATED_APP}-${REPLICATED_CHANNEL}.tar.gz -o ${REPLICATED_APP}-${REPLICATED_CHANNEL}.tar
+```
+**Note**: There is a bug in kURL which incorrectly uses `.tar.gz` extension for a `.tar` file, hence the `-o *.tar`.
+
+* `scp` the `.tar` file from the jumpbox to the airgapped node.
+```shell
+export AIRGAP_PRIVATE_IP=<airgap_private_ip>
+scp ${REPLICATED_APP}.tar ${AIRGAP_PRIVATE_IP}:~
+```
+
+* Untar bundle
+```shell
+export REPLICATED_APP=<app_slug>
+tar -xvf ${REPLICATED_APP}.tar
+```
+
+* Install bundle with `airgap` flag.
+
+```shell
+cat install.sh | sudo bash -s airgap
+```
+
+* Access KOTS Adm via `http://<airgap_private_ip>:8800`.
+
+* If you do not have direct access to the airgap private IP, you can access it via SSH tunnel using the jumpbox's public IP.
+
+```shell
+export AIRGAP_PRIVATE_IP=<airgap_private_ip>
+export JUMPBOX_PUBLIC_IP=<jumpbox_public_ip>
+ssh -N -L 8800:${AIRGAP_PRIVATE_IP}:8800 ${JUMPBOX_PUBLIC_IP}
+```
+
+* * *
+
+## Existing Kubernetes Cluster
+
+Installing into an airgapped existing kubernetes cluster is a similar experience to an [online existing cluster](/vendor/guides/existing-cluster) install. Only difference is you need to ensure the network where the kubernetes cluster is deployed to does not have outbound connectivity to the internet (`0.0.0.0/0`).
```

Comment:
I don't think its worth explaining how to make an airgapped kubernetes cluster, that feels outside the the scope of this guide (unless we're going to explain how to set that up in a particular cloud provider, which I don't think we should)

```suggestion
Installing into an airgapped existing kubernetes cluster is a similar experience to an [online existing cluster](/vendor/guides/existing-cluster) install.
```